text/0000-compat-math-identifiers.md

```diff
@@ -111,15 +115,26 @@ Note that Unicode specifically [mentions Rust as a positive industry example](ht
 
 * Some people might find it difficult to read superscript and subscript letters on lower resolution screens or when using small font sizes.
 
-* Rust might want to decide in the future to give certain superscripts and subscripts syntactic meaning. For example they might want to interpret `a²` as `a * a` or `a₁` as `a[0]`. The latter sounds espcially unlikely though due to the general disagreement of 0-based vs 1-based indexing.
-
 # Rationale and alternatives
 [rationale-and-alternatives]: #rationale-and-alternatives
 
 If this RFC is not implemented then everyone has to keep using ASCII characters for identifier in scientific code, for example `gradient_energy` or `a_12`.
 
 The impact of not implementing it should be fairly small, but implementing it could invite more scientific oriented people to the Rust language and make it easier for them to implement complex concepts.
 
+Alternatively Rust could decide to give the proposed characters syntatic meaning.
+
+Superscript characters could be interpreted as potentiation, for example `let a = 2; let b = a²;` could be a synonym to `let a = 2; let b = a * a;`.
+This would open up a host of questions and potential issues, like:
+- Should `a²⁻³` be interpreted as `1/a`?
+- There is no superscript character for multiplication `*`.
+
+`∞` could be a synonym or replacement to `f32::INIFITY`, however there is no precedence for using non-ASCII characters in `core`/`std` and this would likely meet considerable opposition.
+
+Derivatives could be added as a language features via auto-differentiation techniques thus giving `∇` and `∂` syntactic meaning, however there is no precedence of this in other languages and similar features are usually provided by libraries.
```

Comment:
there is [experimental support for automatic differentiation](https://github.com/rust-lang/rust/issues/124509) being worked on for rustc.
Mathematica supports the syntax $∂_{x}f$ for $\frac{∂f}{∂x}$ and the syntax $∇_{x}f$ for the gradient of $f$ with respect to $x$.